connector/connect/docs/client-connection-string.md

```diff
@@ -81,6 +82,15 @@ sc://hostname:port/;param1=value;param2=value
     <pre>user_id=Martin</pre>
     </td>
   </tr>
+  <tr>
+    <td>user_agent</td>
+    <td>String</td>
+    <td>The user agent acting on behalf of the user, typically applications
+    that use Spark Connect to implement its functionality and execute Spark
+    requests on behalf of the user.<br/>
+    <i>Default: </i><pre>_SPARK_CONNECT_PYTHON</pre> in the Python client</td>
```

Comment:
I see that this is changed to default value in Python client. 

This is fine. Probably in the future we just list the default value for the Scala client.